cloudify_azure/utils.py

```diff
@@ -440,7 +440,9 @@ def handle_properties_keys(data):
     return cleanup_empty_params(data)
 
 
-def get_client_config(properties):
+def get_client_config(properties, other_properties=None):
+    other_properties = other_properties or {}
+    properties.update(other_properties)
```

Comment:
I prefer that we will override other properties with properties. Order of importance is:
1. Client Config from Node Props
2. Client Config from Plugin Props
That means that we first get plugin props client config and then if there is a client config from node props, we override the first with that.

```suggestion
    other_properties.update(properties)
```